proto/blueprint.proto

```diff
@@ -115,6 +115,8 @@ message List {
   repeated int32 adverts = 11;
   optional MyGuardianFollow my_guardian_follow = 12;
   string id = 13;
+  optional string web_uri = 14;
```

Comment:
In the MAPI codebase, we don't always return a web uri for a list. Hence this is optional. A good example of this can be seen on our [Section endpoint](https://github.com/guardian/mobile-apps-api/blob/003f7af30213d62fb9c638c325718c6e9a81cf16/mobile-fronts/app/controllers/lists/declarations/SectionList.scala#L43). I'm not entirely sure why this is. @aoifemcl15 @BenSullivan-iOS is this going to be an issue?

Reply:
Good stuff - however, do you think this provides a good opportunity to instead let MAPI provide you a URI? And then you can delete that fallback?

Reply:
Yeah good idea - the fallback we generate isn't actually a valid url, eg. ""https://www.theguardian.com/uk/lists/section/law". 
Ideally it would be valid and match the web url for the relevant section, eg. "https://www.theguardian.com/law" but not sure how feasible that would be on the MAPI side?

Reply:
On second thoughts, I wonder given the `webUri` has a specific use case here for tracking - should it be included as part of the tracking object? That way, even if an invalid url was used - at least there would be no risk of the url being used for other purposes?

Reply:
@aoifemcl15 good point. Hm, it does seem a bit too general to perhaps include in the tracking object? If we ended up having to move it out of the object we'd either need to duplicate or make a breaking change. Maybe a good compromise is a comment above the field just saying this we believe this is only used for tracking. What do you think? I'm not precious either way.

I'm going to investigate creating a section URI via MAPI now.

Reply:
Yeah I'm for it to live outside of the tracking object as long as it's a valid web url. In the case of sections - if MAPI is unable to return the corresponding web url, maybe we can leave the fallback mechanism to the native app?